charts/gsp-cluster/templates/02-gsp-system/service-operator/service-account.yaml

```diff
@@ -4,6 +4,8 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ template "serviceOperator.serviceAccountName" . }}
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::{{ .Values.global.account.id }}:role/{{ .Values.global.cluster.name }}-service-operator
```

Comment:
I think we should pass the whole role ARN into the chart, rather than constructing it in a way that magically matches how terraform happens to do it.